spec/services/processors/blog_post_views_updater_spec.rb

```diff
@@ -0,0 +1,78 @@
+require 'rails_helper'
+
+describe Processors::BlogPostViewsUpdater do
+  describe '.call' do
+    let(:blog_post) { create(:blog_post, published_at: publish_date) }
+    let(:blog_post_views_payload) do
+      create(:blog_post_views_payload, publish_datetime: publish_date).with_indifferent_access
+    end
+    let(:publish_date) { Time.zone.now.beginning_of_month }
+    let(:current_month_views) do
+      blog_post_views_payload['years'][publish_date.year.to_s]['months'][publish_date.month.to_s]
+    end
+    let(:api_service) { instance_double(WordpressService) }
+
+    before do
+      allow_any_instance_of(described_class).to receive(:wordpress_service).and_return(api_service)
+      allow(api_service)
```

Comment:
It's a not something that needs to be fixed now, but I usually prefer to use Webmock than to stub services. I'm interested in rest of the team's opinion on this as well.

Reply:
Yeah, I agree with that! I'll mock the call instead

Reply:
I finally added it in a separate PR: https://github.com/rootstrap/rs-code-review-metrics/pull/169